ui/src/views/Releases/ReleaseV2/index.jsx

```diff
@@ -0,0 +1,346 @@
+              products.data && getSuggestions(products.data.data.product.sort())
+            }
+            label="Product"
+            required
+            disabled={!isNewRelease}
+            inputProps={{
+              autoFocus: true,
+              fullWidth: true,
+            }}
+          />
+          <br />
+          <br />
+          <div className={classes.uploadReleaseDiv}>
+            <label htmlFor="upload-release-file">
+              <input
+                disabled={isReadOnly}
+                accept=".json"
+                className={classes.inputOfTypeFile}
+                id="upload-release-file"
+                type="file"
+                onChange={handleUploadRelease}
+              />
+              <Button
+                disabled={isReadOnly}
+                size="small"
+                variant="outlined"
+                component="span"
+                className={classes.uploadReleaseButton}>
+                Upload Release
+              </Button>
+            </label>
+          </div>
+          <CodeEditor
+            onChange={handleReleaseEditorChange}
+            value={releaseEditorValue}
+            readOnly={isReadOnly}
+          />
+          <br />
+          <br />
+          <Fragment>
+            <Tooltip title={isNewRelease ? 'Create Release' : 'Update Release'}>
+              {/* Add <div /> to avoid material-ui error "you are providing
+              a disabled `button` child to the Tooltip component." */}
+              <div
+                className={classNames(
+                  classes.fabWithTooltip,
+                  classes.saveButton,
+                  {
+                    [classes.secondFab]: hasScheduledChange && !isReadOnly,
+                    [classes.fab]: !hasScheduledChange || isReadOnly,
+                  }
+                )}>
+                <Fab
+                  disabled={!user || actionLoading || isReadOnly}
+                  onClick={
+                    isNewRelease ? handleReleaseCreate : handleReleaseUpdate
+                  }
+                  color="primary">
+                  <ContentSaveIcon />
+                </Fab>
+              </div>
+            </Tooltip>
+            {hasScheduledChange && !isReadOnly && (
+              <SpeedDial
+                FabProps={{
+                  disabled:
+                    !user || actionLoading || isReadOnly || !hasScheduledChange,
+                }}
+                ariaLabel="Secondary Actions">
+                <SpeedDialAction
+                  icon={<DeleteIcon />}
+                  tooltipOpen
+                  tooltipTitle="Cancel Pending Change"
+                  onClick={handleScheduledChangeDelete}
+                />
+              </SpeedDial>
+            )}
+          </Fragment>
+          <Snackbar onClose={handleSnackbarClose} {...snackbarState} />
+        </Fragment>
+      )}
+    </Dashboard>
+  );
+}
+
+export default withUser(ReleaseV2);
```

Comment:
Will this be renamed to `Releases/Release/index.jsx` once v2 is stable? Also, do we know how long before that would happen? Should we delete the content of `Releases/Release/index.jsx` if it's no longer used?

Reply:
Yeah, I'm going to remove all of the `V2` references after `V1` Release stuff goes away.

And yes! We can kill `Releases/Release/index.jsx` already.

Reply:
Awesome. Thanks for clarifying.